ansible_builder/steps.py

```diff
@@ -21,8 +21,11 @@ def __init__(self, additional_steps):
         elif isinstance(additional_steps, list):
             lines = additional_steps
         else:
-            print(MessageColors.FAIL + "Error: Unknown type found for additional_build_steps; "
-                  "must be list or multi-line string." + MessageColors.ENDC)
+            raise DefinitionError(
+                """
+                Error: Unknown type found for additional_build_steps; must be list or multi-line string.
+                """
+            )
```

Comment:
I don't think you necessarily need to keep this conditional, because it should be covered by the validate method.

But if you do, feel free to introduce new module `ansible_builder.exceptions` and put the definition of `DefinitionError` there so that you eliminate any possibility of circular imports. You really want to keep python imports without loops, and in-line imports are something you should avoid whenever you have the freedom to avoid them.

Reply:
Makes sense, I'll do that!  

Just noticed I forgot to remove the unused `from .colors import MessageColors` from `steps.py` as well, so I'll be pushing another commit shortly.